examples/job_crud.py

```diff
@@ -54,6 +56,19 @@ def create_job(api_instance, job):
         body=job,
         namespace="default")
     print("Job created. status='%s'" % str(api_response.status))
+    print("Job status='%s'" % str(get_job_status(api_instance)))
```

Comment:
better to move the print to the polling loop below, so that every 1s, the status is printed out

Reply:
@yliaog Do you have any idea why the travis-ci build is failing?